testenv/testenv.properties

```diff
@@ -9,11 +9,11 @@ OPENJ9_SYSTEMTEST_BRANCH=openj9-0.41.0
 ADOPTOPENJDK_SYSTEMTEST_REPO=https://github.com/adoptium/aqa-systemtest.git
 ADOPTOPENJDK_SYSTEMTEST_BRANCH=v0.9.9
 JDK8_REPO=https://github.com/adoptium/jdk8u.git
-JDK8_BRANCH=jdk8u392-b07
+JDK8_BRANCH=jdk8u392-b08
 JDK11_REPO=https://github.com/adoptium/jdk11u.git
-JDK11_BRANCH=jdk-11.0.21+8
+JDK11_BRANCH=jdk-11.0.21+9
 JDK17_REPO=https://github.com/adoptium/jdk17u.git
-JDK17_BRANCH=jdk-17.0.9+7
+JDK17_BRANCH=jdk-17.0.9+9
 JDK21_REPO=https://github.com/adoptium/jdk21u.git
 JDK21_BRANCH=jdk-21.0.1
```

Comment:
@jerboaa Do you have a idea about the jdk21 GA tag? Also for other versions?

Reply:
@sophia-guo JDK 21 is still maintained by Oracle. There is no way of knowing until the `jdk-21.0.1-ga` tag is being pushed upstream.

Reply:
Other versions look right to me.

Reply:
For jdk21 we will make a guess based on @andrew-m-leonard suggestion. If it's wrong we can update later.